ts/DataGrid/Header/HeaderRow.ts

```diff
@@ -46,51 +44,142 @@ class HeaderRow extends Row {
     *
     * */
 
-    /**
-     * Constructs a row in the data grid header.
-     *
-     * @param viewport
-     * The Data Grid Table instance which the row belongs to.
-     */
-    constructor(viewport: Table) {
-        super(viewport);
-        this.setRowAttributes();
-    }
-
-
     /* *
     *
     *  Methods
     *
     * */
 
-    public override createCell(column: Column): Cell {
+    public override createCell(column: Column): HeaderCell {
         return new HeaderCell(column, this);
     }
 
     /**
      * Renders the row's content in the header.
+     *
+     * @param level
+     * The current level in the header tree
      */
-    public override render(): void {
+    public renderMultipleLevel(level: number): void {
+        const header = this.viewport.dataGrid.userOptions?.settings?.header;
+        const vp = this.viewport;
+        const enabledColumns = vp.dataGrid.enabledColumns;
 
         // Render element
-        this.viewport.theadElement.appendChild(
+        vp.theadElement.appendChild(
             this.htmlElement
         );
 
         this.htmlElement.classList.add(
             Globals.classNames.headerRow
         );
 
-        super.render();
+        if (header) {
+            const columnsOnLevel = this.getColumnsAtLevel(header, level);
```

Comment:
1. In the case of the lowest level, do we also call everything that is in the body of this if? I think we can also call `super.render()` here; unless I'm mistaken.

2. Can we get it from this level? Eg.:

    ```ts
    if (!header) {
      super.render();
      this.reflow();
      return;
    }
    
    [...]
    
    this.reflow();
    ```